src/java.base/share/classes/sun/security/provider/SunEntries.java

```diff
@@ -109,6 +111,51 @@ public final class SunEntries {
         // common attribute map
         HashMap<String, String> attrs = new HashMap<>(3);
 
+        /*
+         * Certificates
+         */
+        addWithAlias(p, "CertificateFactory", "X.509",
+                "sun.security.provider.X509Factory", attrs);
```

Comment:
The attributes are different than before and different than in jdk11. For example, for `CertificateFactory`, they include `ImplementedIn=Software`.

Reply:
added